tester_app/config/initializers/react_on_rails.rb

```diff
@@ -0,0 +1,21 @@
+# Shown below are the defaults for configuration
+ReactOnRails.configure do |config|
```

Comment:
Sure -- change the comments around the options

Not for helper:

+  # Server rendering:
+  # Server bundle is a single file for all server rendering of components.
+  # If you wish to use render_js in your views without any file, set this to "" to avoid warnings.
+  config.server_bundle_js_file = "app/assets/javascripts/generated/server-bundle.js" # This is the default
+  config.server_renderer_pool_size  = 1   # increase if you're on JRuby
+  config.server_renderer_timeout    = 20  # seconds